Applications/IntelliDivide/Documentation/Statistics/Material/MaterialStatistics-Excel.md

```diff
@@ -0,0 +1,58 @@
+# Anaylze intelliDivide Material Statistics data using Excel
+
+Microsoft Excel is a powerful tool for analyzing and visualizing data. 
+
+<strong>To analyze the data the following steps are necessary:</strong>
+
+Define a new web data source.
+
+![alt text](MaterialStatistics-Excel-01.jpg)
+
+Enter the URL for the Material Statistics API, as described on the [Material Statistics page](MaterialStatistics.md) (e.g. https://connect.homag.cloud/api/intelliDivide/statistics/material?take=1000). 
+
+![alt text](MaterialStatistics-Excel-02.jpg)
+
+Enter Subscription Id (1) and Authentication Key (2): You can find the Subscription Id and Authentication Key on the [Authorization](../../Authorization/Authorization.md) page. Copy and paste these values into the appropriate fields.
+
+![alt text](MaterialStatistics-Excel-03.jpg)
+
+Convert the column to a table.
+
+![alt text](MaterialStatistics-Excel-04.jpg)
+
+Expand the table columns.
+
+![alt text](MaterialStatistics-Excel-05.jpg)
+
+Change the data types of the columns.
```

Comment:
add here a comment about the local. Data is coming from the cloud in "neutral" culture. Dependign on the culture of the receiving excel/powerBI it might be interpreted wrong.